.circleci/config.yml

```diff
@@ -65,6 +65,29 @@ jobs:
           name: Flake8
           command: git submodule sync --recursive && git submodule update --recursive --init && flake8 sofar tests
 
+  test_examples:
+    parameters:
+      version:
+        description: "version tag"
+        default: "latest"
+        type: string
+    executor:
+      name: python-docker
+      version: <<parameters.version>>
+
+    steps:
+      - checkout
+      - python/install-packages:
+          pkg-manager: pip
+          # app-dir: ~/project/package-directory/  # If you're requirements.txt isn't in the root directory.
+          pip-dependency-file: requirements_dev.txt
+      - run:
+          name: Examples
+          command: |
+            pip install -e .
```

Comment:
I assume not. I copy pasted from spharpy. But it doesn't hurt either, or does it?

Reply:
I chose it as it simply does a symlink instead of copying the data. But it should not make a difference. The content of the container gets erased afterwards anyways, so I did not see the need to do a proper install.